src/mono/netcore/sample/wasm/browser-profile/Makefile

```diff
@@ -0,0 +1,33 @@
+TOP=../../../../../..
+DOTNET=$(TOP)/dotnet.sh
+
+ifeq ($(V),)
+DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
+else
+DOTNET_Q_ARGS=--nologo
+DOTNET_MONO_LOG_LEVEL=--setenv=MONO_LOG_LEVEL=debug
+endif
+
+CONFIG?=Release
+
+build-aot-profile:
+	EMSDK_PATH=$(realpath $(TOP)/src/mono/wasm/emsdk) $(DOTNET) publish $(DOTNET_Q_ARGS) /p:EnableProfiler=true /p:WasmBuildNative=true /p:Configuration=$(CONFIG) /p:TargetArchitecture=wasm /p:TargetOS=Browser $(MSBUILD_ARGS) WasmSample.csproj
+
+run-aot-profiled:
+	cp $(HOME)/Downloads/data.aotprofile ./bin/$(CONFIG)/AppBundle
```

Comment:
Instead of hardcoding the path, use a var here, and error out if that isn't given.